tests/templates/kuttl/tls/21-install-requirements.yaml.j2

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: kubectl cp -n $NAMESPACE ./check-tls.py  trino-test-helper-0:/tmp
+  - script: kubectl cp -n $NAMESPACE ./test-config.yaml  trino-test-helper-0:/tmp
+  - script: kubectl exec -n $NAMESPACE trino-test-helper-0 -- pip install -q trino pyyaml
```

Comment:
Need to lock the versions here.

Reply:
Could we dump it out as `ini` or `json` or something instead and use the built-in `configparse`/`json` modules?

Reply:
Ill add a requirements.txt as we do in the other operators.

Reply:
As far as I can tell, we only read the YAML in the test script anyway, so there's no reason not to use a format that Python has built-in support for instead.

Reply:
switched to json